exec/cnex/test_string_support.c

```diff
@@ -0,0 +1,82 @@
+#include <assert.h>
+#include <string.h>
+
+#include "nstring.h"
+
+
+int64_t string_findCString(TString *self, const char *p)
+{
+    int64_t r = -1;
+
+    TString *psz = string_createCString(p);
```

Comment:
Also here you don't have to allocate a new string. Use a local variable.

Reply:
Ok, but I have to cast away the constness of p to be able to do it.  That, or change the function prototype.  I don't know which one is more appropriate.